File: Tutorial_Sample_Code/Other/CRUFL_ExchangeRate/CS/CS_Win_UseExchangeRate/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using CrystalDecisions.CrystalReports.Engine;
using CrystalDecisions.Shared;
using System.IO;


namespace CS_Win_UseExchangeRate
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private ReportDocument functionTestReport;

        private void ConfigureCrystalReports()
        {
            functionTestReport = new ReportDocument();
            string reportPath = new DirectoryInfo(Application.StartupPath).Parent.Parent.FullName;
            functionTestReport.Load(reportPath + "\\FunctionTest.rpt");
            crystalReportViewer1.ReportSource = functionTestReport;
        }


        private void Form1_Load(object sender, EventArgs e)
        {
            ConfigureCrystalReports();
        }
    }
}

File: Tutorial_Sample_Code/Other/CRUFL_ExchangeRate/VB/VB_Win_UseExchangeRate/Form1.vb
Imports CrystalDecisions.CrystalReports.Engine
Imports CrystalDecisions.Shared


Public Class Form1

    Private functionTestReport As ReportDocument

    Private Sub ConfigureCrystalReports()
        functionTestReport = New ReportDocument()
        functionTestReport.Load((IO.Directory.GetParent(IO.Directory.GetParent(Application.StartupPath).ToString).ToString) & "\FunctionTest.rpt")
        CrystalReportViewer1.ReportSource = functionTestReport
    End Sub

    Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        ConfigureCrystalReports()
    End Sub

End Class
